Makefile

```diff
@@ -91,6 +92,7 @@ version:
 .PHONY: docker_run
 docker_run:
 	docker run --rm -it \
+		--user ${KITCHEN_USER} \
```

Comment:
Thinking more about this, the issue I faced on my laptop with the credentials was more about using symlinks instead of hardlinks. Either kitchen (or the mount namespace maybe) didn't like that.

Hardlinking the key removes the need for this. I'll revert those changes.